### PR TITLE
CTA and Materials sections

### PR DESCRIPTION
**Pull request by nbond211**
Screen Shot 2019-04-09 at 7 52 16 PM
Screen Shot 2019-04-09 at 7 52 37 PM
Screen Shot 2019-04-09 at 7 52 52 PM

### CODE REVIEW

File: src/components/button.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import styles from "../styles/button.module.css"
+
+export default function PhotoStrip(props) {
```
Comment (maryats) [inline src/components/button.js]: This component should probably be called Button haha

**Review by maryats — APPROVED** [review]
👌

File: src/components/button.js (hunk 1 above)
Comment (nbond211) [reply src/components/button.js]: 🙃

**Comment by nbond211** [comment]
Screen Shot 2019-04-10 at 10 56 07 AM
Screen Shot 2019-04-10 at 10 56 30 AM
Screen Shot 2019-04-10 at 10 57 10 AM

**Comment by maryats** [comment]
I'm not sure the yellow background should overlap behind the Testimonials section on tablet; can probably adjust that later though

**Pull request merged**